### DISCUSSION TITLE
Questions for ivan-developer-01

### DISCUSSION THREAD

**Discussion by ivan-developer-01** [ivan-developer-01/ivan-developer-01#1] [General]
Ask questions for me here.

*I am leaving the right for me to not answer the questions when I don't want.*

I am open for ideas.

**Comment by ivan-developer-01** [comment]
It began!

> ![Requiring 2FA](https://github.com/ivan-developer-01/ivan-developer-01/assets/127977316/66b7d327-1b99-4bdf-901c-672671738deb)

**Reply by 5irehackerthon** [reply]
Hello Ivan
I would like to Start learning frontend web development but l don't know where to start as well as getting contents

**Reply by ivan-developer-01** [reply]
Hello :wave: @5irehackerthon!  
Personally I am learning on paid courses (and they are, well, a bit overpriced but that's not the point). But I can recommend:

- MDN Web docs https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/HTML_basics - a great documentation where you can learn too;
- freeCodeCamp https://www.freecodecamp.org/learn/ - I recommend passing first three curriculums for frontend range;
- JavaScript Tutorial - https://javascript.info (great tutorial for JavaScript but only when you know HTML first)
- YouTube (yes, I passed my very first curriculum there!)

And there is very many tutorials.  
The list I wrote above is in the order I recommend to pass. If you have any other questions, feel free to ask!

**Comment by ivan-developer-01** [comment]
Historical moment!

![](https://github.com/ivan-developer-01/ivan-developer-01/assets/127977316/f2a818f8-db1b-4030-8c10-5a5a39565c0f)

**Reply by ivan-developer-01** [reply]
Another historical moment.

![888](https://github.com/ivan-developer-01/ivan-developer-01/assets/127977316/b0df65e3-ee64-4d1c-8db2-c5d029c2f9d8)

**Reply by ivan-developer-01** [reply]
Yet another historical moment.

![1024 contributions in the last year](https://github.com/ivan-developer-01/ivan-developer-01/assets/127977316/569c14f2-3c99-456a-b531-b9649c11f4c9)

**Reply by ivan-developer-01** [reply]
![](https://github.com/ivan-developer-01/ivan-developer-01/assets/127977316/d8d5924e-9d07-4383-b365-cd616396ebb1)
